### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by AlexDarchuk**

### CODE REVIEW

File: homework-1/task-1/main.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const inputNumber = document.querySelector('.number');
+const stairsBtn = document.querySelector('.stairs-btn');
+const showStairs = document.querySelector('.show-stairs');
+const lastStairs = document.querySelector('.lastStairs');
+
+let str = '$';
```
Comment (Maxim-Bondar) [inline homework-1/task-1/main.js]: to common name, change to currencySymbol

File: homework-1/task-2/main.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+function topRandomDomino () {
+    let topDominoArray = [];
+    for(let i = 0; i < 6; i++) {
+        topDominoArray.push(Math.floor(Math.random() * (6 - 1) + 1));
+    }
+    return topDominoArray;
+}
+
+function bottomRandomDomino () {
```
Comment (Maxim-Bondar) [inline homework-1/task-2/main.js]: this two methods are equal, so you don't need to dublicat it , just create only one helper

File: homework-1/task-3/main.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    100: 5,
+    50: 34,
+    20: 10,
+    10: 10,
+};
+
+
+let quantity = Object.entries(limits);
+
+quantity.forEach((value) => {
```
Comment (Maxim-Bondar) [inline homework-1/task-3/main.js]: don't understand what are you doing here

File: homework-1/task-3/main.js [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    let p = document.createElement('p');
+    p.innerHTML = `В банкоматі такі купюри ${key} в кількості ${limits[key]} = ${sum}`;
+    bank.append(p);
+}
+
+totalMoney.innerHTML = `Загальна сума грошей ${moneyTotal - moneyInput.value}`;
+
+
+
+let iWantToGet = (ammountRequired, limits) => {
```
Comment (Maxim-Bondar) [inline homework-1/task-3/main.js]: Where you have found this solution?

File: homework-1/task-1/main.js (hunk 1 above)
Comment (AlexDarchuk) [reply homework-1/task-1/main.js]: i think done)))

File: homework-1/task-2/main.js (hunk 2 above)
Comment (AlexDarchuk) [reply homework-1/task-2/main.js]: done!

File: homework-1/task-3/main.js (hunk 3 above)
Comment (AlexDarchuk) [reply homework-1/task-3/main.js]: function of calculating the total amount of currency....maybe there is an easier way

File: homework-1/task-3/main.js (hunk 4 above)
Comment (AlexDarchuk) [reply homework-1/task-3/main.js]: in internet...but the show in page, and counting, change in the amount of currency total amount is i did myself

**Pull request merged**